node-fs-writefile/cp.js

```diff
@@ -0,0 +1,20 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+const [, , origFile, newFile] = process.argv;
+
+if (!origFile.endsWith('.txt')) {
+  console.error('Source file must be .txt format');
+  process.exit(1);
+}
+if (!newFile.endsWith('.txt')) {
+  console.error('Destination file must be .txt format');
+  process.exit(1);
+}
+
+readFile(origFile, 'utf8')
+  .then((value) => {
+    writeFile(newFile, value)
+      .then((value) => value)
```

Comment:
This doesn't do anything, so it's fine to leave it off. Promises are not _required_ to have `.then` handlers.